strongbox-resources/strongbox-storage-api-resources/pom.xml

```diff
@@ -45,6 +45,16 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-resources-plugin</artifactId>
+                <configuration>
+                    <useDefaultDelimiters>false</useDefaultDelimiters>
+                    <delimiters>
+                        <delimiter>@{*}</delimiter>
```

Comment:
Just out of curiousity and for my better understanding, why are you changing the delimiter? Is the `$` sign a special character in YAML?

Reply:
This delimiter is used to make reference to a maven POM property and avoid conflicts with the "${*}" deliimter. [Link](https://docs.spring.io/spring-boot/docs/current/reference/html/howto-properties-and-configuration.html).